feature-libs/product-configurator/rulebased/components/attribute/quantity/configurator-attribute-quantity.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@angular/core';
+import { Configurator } from '../../../core/model/configurator.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ConfiguratorAttributeQuantityService {
+  /**
+   * Checks if the interaction with the quantity control needs
+   * to be disabled
+   * @param value Selected value
+   * @returns Quantity actions disabled?
+   */
+  disableQuantityActions(value: any): boolean {
```

Comment:
can it be this: `disableQuantityActions(value?: string)`? If possible, we try to reduce the usage of `any`.